documentation/migration.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Migration
```

Comment:
Maybe the title should be "Migration to Unexpected 11"? Or maybe it would be better to also list migration guides for future versions here, but then the heading below (line 8) should be "Migration to Unexpected 11" or similar.